lib/utils/__tests__/isStandardSyntaxHexColor.test.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const isStandardSyntaxHexColor = require('../isStandardSyntaxHexColor');
+
+describe('isStandardSyntaxHexColor', () => {
+	it('default hex', () => {
+		expect(isStandardSyntaxHexColor('#000000')).toBeTruthy();
+	});
+	it('less map usage', () => {
+		expect(isStandardSyntaxHexColor('#prop[someprop]')).toBeFalsy();
```

Comment:
[suggestion] For more accuracy, how about using `toBe()` like this?

```suggestion
		expect(isStandardSyntaxHexColor('#000000')).toBe(true);
	});
	it('less map usage', () => {
		expect(isStandardSyntaxHexColor('#prop[someprop]')).toBe(false);
```